xrpl_types/src/amount.rs

```diff
@@ -40,6 +40,38 @@ impl Amount {
         }
     }
 
+    /// Try to construct an amount from a 'spec' string.
+    /// Examples:
+    /// Amount::try_from_str("26.231 USD:rhub8VRN55s94qWKDv6jmDy1pUykJzF3wq");
```

Comment:
It this an "official" format or a format specific to the cli application. If it is the latter, then the logic should be in the `xrpl_cli` crate?

Reply:
At least I have not seen it used in the XRPL docs?